src/core/positioning/egenioussreceiver.h

```diff
@@ -21,11 +25,10 @@ class EgenioussReceiver : public AbstractGnssReceiver
   private slots:
     void onReadyRead();
     void handleError( QAbstractSocket::SocketError error );
+    void setSocketState( const QAbstractSocket::SocketState socketState );
 
   private:
     void processReceivedData();
-    void connected();
-    void disconnected();
 
   private:
     QTcpSocket *mTcpSocket = nullptr;
```

Comment:
mTcpSocket should not be  a pointer. Now its a memory leak as it is never deleted